src/components/index.ts

```diff
@@ -74,8 +74,5 @@ export { ContentPage } from './ContentPage'
 export { PricingPlanProperty, PricingPlanFeature, PricingPlan } from './Pricing'
 export type { Features } from './Pricing'
 
-// Webinars
-export { WebinarLayout } from './WebinarLayout'
-
-// Guides
-export { GuideLayout } from './GuideLayout'
+// Webinars, Guides, Reports
+export { SimpleGatedPgLayout } from './SimpleGatedPg'
```

Comment:
How about naming it `GatedResourceLayout` since it gates a resource, and then placing this under Layouts under line 71?

```suggestion
export { GatedResourceLayout } from './GatedResourceLayout'
```